oximeter/db/src/client.rs

```diff
@@ -503,10 +516,12 @@ impl Client {
         response
     }
 
-    async fn get_schema(&self) -> Result<(), Error> {
+    async fn get_schema_locked(
```

Comment:
I find this name a little confusing since the function doesn't seem to itself have anything to do with locking, but rather assumes the passed in schema is locked. Is that right? Maybe a comment to clarify if so, if not a name change.

Reply:
Sure, I can do that. I think that's a Rust-ism, used when the method can only be called after acquiring a lock, in this case around the schema map.

Reply:
Done in a67f22617